Makefile

```
@@ -0,0 +1,36 @@
+# Variables
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)